packages/components/src/utils.ts

```diff
@@ -673,3 +673,15 @@ export const convertBaseMessagetoIMessage = (messages: BaseMessage[]): IMessage[
     }
     return formatmessages
 }
+
+/**
+ * Convert MultiOptions String to String Array
+ * @param {string} inputString
+ * @returns {string[]}
+ */
+export const convertMultiOptionsToStringArray = (inputString: string): string[] => {
+    let ArrayString: string[] = []
+    if (inputString) ArrayString = JSON.parse(inputString)
```

Comment:
oops, I meant the other way around. this would enable us to catch all misinformed JSON as well, in addition to null/undefined/empty...

try {
ArrayString = JSON.parse(....)
}catch(Error e) {
ArrayString = []
}